src/decorator-observable.js

```diff
@@ -9,6 +9,11 @@ export function observable(targetOrConfig: any, key: string, descriptor?: Proper
 
     // use a convention to compute the inner property name
     let innerPropertyName = `_${key}`;
+    const innerPropertyDescriptor: PropertyDescriptor = {
+      configurable: true,
+      enumerable: false,
+      writable: true,
+    };
```

Comment:
Sorry for my confusing comment above. Removing [this](https://github.com/aurelia/binding/pull/506/files#diff-26d3f601c99bc9cd08a30349643a0ac2R44) line causes some old, already existing test to fail.